app/models/lightweight_activity.rb

```diff
@@ -138,7 +138,8 @@ def export
                                         :editor_mode,
                                         :student_report_enabled,
                                         :show_submit_button,
-                                        :runtime ])
+                                        :runtime,
+                                        :background_image ])
```

Comment:
It looks like this background_image property is not imported correctly. I'm not sure but I think the correct way to do that is to add it to the extract_from_hash method.